### ISSUE TITLE
addition to Gemfile brokes Rails 5 API default response of jbuilder

### ISSUE THREAD

**Issue by Pepan** [gonzalo-bulnes/simple_token_authentication#278]
Hi, 
when I add this gem to my Gefile, Raisl (in API mode) start respond 200 on every get and never perform action.json.builder

When I remove it from Gemfile it magicaly starts wotking again.

Workaround is to add respond_with @object to controller action.

like:
```ruby
def index
      @games = Game.all
      respond_with @games
end
```

**Comment by gonzalo-bulnes** [comment]
Hello @Pepan,

Can you give me more details please? Or is there any example I could look at to reproduce the issue?

I can't think of anything that would explain that behaviour at first sight, but it is curious enough to investigate further.

Thanks!

**Comment by Pepan** [comment]
Hi, I made short video.
http://screencast.com/t/RobsubpjVX7
and attached zipped demo dir
[Rails5apidemo.zip](https://github.com/gonzalo-bulnes/simple_token_authentication/files/575105/Rails5apidemo.zip)

**Comment by gonzalo-bulnes** [comment]
Awesome @Pepan thank you! I'll take a look at this as soon as I can : )

**Comment by templeman15** [comment]
@gonzalo-bulnes Any ETA for getting this fixed? Thanks!

**Comment by gonzalo-bulnes** [comment]
Hello @templeman15, contributions are welcome!

**Comment by IvRRimum** [comment]
+1 have the same problem!

**Comment by IvRRimum** [comment]
Debuged further, found that rails_api adapter is the one who messes up the view rendering. Investigating further.

**Comment by IvRRimum** [comment]
Okey, i got it working.. but its really odd. So when i disable rails_api adapter it render jbuilder fine. And if i add `acts_as_token_authentication_handler_for User` to the controller it works as expected showing me `Please sign in` stuff.

```
Finished in 0.28999 seconds (files took 1.02 seconds to load)
244 examples, 0 failures
```

It seems really fishy, but whatever. I will make a merge request and we can discuss it there.

**Comment by gonzalo-bulnes** [comment]
Just to sum up the discussion in https://github.com/gonzalo-bulnes/simple_token_authentication/issues/287:

> `ActionController::Base` does depend on `ActionController::Metal` [since Rails 4.1.0][src].
And `ActionController::API` does [since Rails 5.0.0][src-api].

Within these version constraints, the Rails and Rails API adapters are not necessary as long as the Rails Metal adapter is loaded.

Disabling the Rails API adapter does fix the interference with _jbuilder_. (The root cause is still unclear, but there is no reason to load adapters that are not necessary, so I'll release the workaround.)



  [src]: https://github.com/rails/rails/commit/d8888b94b32b28003ec247d0565e73dee731275e#diff-0d109f83f6a53964b7649397b1c65c6dR163
  [src-api]: https://github.com/rails/rails/commit/032778eefb4439a72c2933ea0bd4a7a0ef776234

**Comment by gonzalo-bulnes** [comment]
I shipped the workaround in the **v1.15.1** release : ) Thank you again @IvRRimum, and @Pepan for providing such a helpful demo app!
